packages/hub/services/gas-estimation.ts

```diff
@@ -76,62 +67,43 @@ export default class GasEstimationService {
   }
 
   private async estimatePaymentExecution(params: GasEstimationParams) {
-    if (
-      !params.tokenAddress ||
-      params.tokenAddress === '' ||
-      !params.gasTokenAddress ||
-      params.gasTokenAddress === ''
-    ) {
-      throw Error(`tokenAddress and gasTokenAddress is required in ${params.scenario}`);
-    }
-
     let provider = this.ethersProvider.getInstance(params.chainId);
     let signer = new Wallet(config.get('hubPrivateKey'));
     let scheduledPaymentModule = await this.cardpay.getSDK('ScheduledPaymentModule', provider, signer);
 
-    // Both transferAmount and gasPrice should be as small as possible
-    // (so that we don't have to keep a non-trivial balance of tokens in the crank)
-    // given their token decimal amount.
-    // For transferAmount we should consider the underflow error,
-    // since transferAmount will be calculated with fee percentage.
-    // And gasPrice is price per unit of gas, so it will be multiplied by the gas execution.
-    let transferAmount = '1000';
-    let gasPrice = '100';
+    // Estimating using transfer and gas amount are zero,
+    // so we don't need to deposit any amount to the safe owned by the crank.
+    // Using USDC token because USDC token doesn't throw any errors
+    // if transfer amount is zero.
+    // Other tokens might revert erorr if the transfer amount is zero.
```

Comment:
```suggestion
  // Payment execution can support multiple transfer and gas tokens, but for estimating the 
  // execution we can simply use the USDC token. This is because this token contract allows 
  // zero amounts for transfer, which is usually not the case for other token implementations.
  // Because of this feature, we don't have to deposit any USDC to the safe if we
  // set the transaction amounts to 0. If we used some other transfer/gas token for estimation,
  // then we would have to deposit an adequate amount of tokens to the safe in order for 
  // the estimation process to work.
```